### PR TITLE
Replaced old documentation with high-level description of the project

### PR DESCRIPTION
**Pull request by valearna**

### CODE REVIEW

**Review by sweng66 — APPROVED** [review]
Thanks @valearna

File: README.md [hunk 1]
```diff
@@ -1,553 +1,14 @@
...
-There are several plugins available see https://sqlalchemy-continuum.readthedocs.io/en/latest/plugins.html
-Also see test_013_reference_cont.py for examples.
\ No newline at end of file
+## Overview
+The Alliance of Genome Resources (AGR) Literature Service is the backend component of the Alliance Bibliography Central
+(ABC) platform. It provides a number of reference management functions, including:
+
+- **Database interface**: Manages the central ABC database and provides interfaces to CRUD operations through RESTful API endpoints
+- **Search**: Creates an elasticsearch index from the database, keeps the data in sync and provides access to the indexed reference data
+- **Literature processing scripts**: Fetches references and other related data from PubMed
```
Comment (azurebrd) [inline README.md:9]: @valearna @sweng66 from PubMed and the mods ?

**Review by azurebrd — DISMISSED** [review]
Looks good, but a question about whether Literature processing should also mention the MODs

File: README.md (hunk 1 above)
Comment (valearna) [reply README.md:9]: yes, good point @azurebrd, I'll update the text !

**Review by azurebrd — APPROVED** [review]
Thanks @valearna !

**Pull request merged**